src/server/api/routers/traces.ts

```diff
@@ -275,4 +275,37 @@ export const traceRouter = createTRPCRouter({
         observations: enrichedObservations as ObservationReturnType[],
       };
     }),
+  delete: protectedGetTraceProcedure
+    .input(z.object({ traceId: z.string() }))
+    .mutation(async ({ input, ctx }) => {
+      const trace = await ctx.prisma.trace.findFirstOrThrow({
```

Comment:
add a check whether user has a sufficient role to perform this action

Reply:
ok... no problem

Reply:
I have checked scopes array, there is no scope for trace:CUD or trace:delete.
I am thinking to create new scopes. Kindly give any suggestion if you have.
If yes, then tell me that should only admin and owner be allowed to delete traces or member can also delete traces?